external-crates/move/crates/move-vm-runtime/src/runtime/package_resolution.rs

```diff
@@ -196,7 +251,7 @@ pub fn jit_and_cache_package(
 //  /// the type cache.
 //  pub fn get_module(
 //      &self,
-//      data_store: &impl DataStore,
+//      data_store: &impl PackageStore,
```

Comment:
Can we just drop these comments finally?

Reply:
I was waffling on this, but yea let's do it. I'll remove them in this PR.